aten/src/ATen/cudnn/Handle.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Why are we changing the name of the file from "Handles.h" to "Handle.h" ?

Reply:
By request from @ezyang